### ISSUE TITLE
API POST commads not response 400 code when ClosedChannelException or  device offline

### ISSUE THREAD

**Issue by vilix13** [tananaev/traccar#3113]
Hello, I use `api/commands` api for send custom commands to FM1120, when I send a command no difference device is connected to the server or not - i get 200 OK response. But in log expected error:
```
2017-04-21 20:02:56  INFO: [BE95D54A] disconnected
2017-04-21 20:09:28  INFO: [BE95D54A] id: 357454073766964, command type: custom sent
2017-04-21 20:09:28 DEBUG: [BE95D54A: 5027 > 46.211.145.68] HEX: 00000000000000100c0105000000086765747665720d0a0100004d36
2017-04-21 20:09:28  WARN: [BE95D54A] error - ClosedChannelException (... < ActiveDevice:65 < BaseProtocol:57 < ActiveDevice:61 < ...)
```
traccar 3.8 and 3.9

Okay, what I expect is not confirmed by the Api documentation, but in docs:

_**Could happen when dispatching to a device that is offline**, the user doesn’t have permission or an incorrect command type for the device._

I set correct for my device status.interval, and when my device offline or unknown, I still do not get 400 code when call this api.

This is a normal behavior or bug in these versions?

**Comment by tananaev** [comment]
Just tested and I get `400 (Bad Request)` when device is offline. Detailed error:
```
Device is not online - RuntimeException (DeviceManager:485 < CommandResource:38 < ...)
```

**Comment by vilix13** [comment]
I have an assumption that this bug is up to version 3.10 and only with Teltonika protocol.

**Comment by tananaev** [comment]
So, does it work fine on 3.11?

**Comment by vilix13** [comment]
@tananaev just now tested on 3.10 - work fine. I think on 3.11 will also be ok. I think this commit fix that: bf3db6b69ebe36038232fee0de93f7cfbfea45a1

**Comment by tananaev** [comment]
Yes, that's likely the case. Closing the issue then.
